packages/noya-state/src/reducers/pointReducer.ts

```diff
@@ -17,7 +17,22 @@ import { ApplicationState, SetNumberMode } from './applicationReducer';
 export type PointAction =
   | [type: 'setPointCurveMode', curveMode: Sketch.CurveMode]
   | [type: 'setPointCornerRadius', amount: number, mode?: SetNumberMode]
-  | [type: 'setPointX' | 'setPointY', amount: number, mode?: SetNumberMode];
+  | [type: 'setPointX' | 'setPointY', amount: number, mode?: SetNumberMode]
+  | [type: 'selectPoint', point: Point, mode: SetNumberMode];
+
+const POINT_RADIUS = 4;
+
+function isPointInRange(point: Point, rawPoint: Point): boolean {
```

Comment:
Should this function be moved to a utility file?

Reply:
Maybe it can go in `pointSelectors`? The most important thing is that we share that `POINT_RADIUS` constant with the rendering code, so that they don't get out of sync

Reply:
isPointInRange and POINT_RADIUS moved to pointSelectors. Constant is now used in` EditablePath` for consistency